dev-lua/lua-openssl/lua-openssl-0.4.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit toolchain-funcs
+
+DESCRIPTION="OpenSSL binding for Lua"
+HOMEPAGE="https://github.com/zhaozg/lua-openssl"
+SRC_URI="https://github.com/zhaozg/lua-openssl/archive/${PV}.tar.gz -> lua-openssl-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+luajit"
+
+RDEPEND="
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? ( >=dev-lang/lua-5.1 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+use luajit && LUAV=luajit || LUAV=lua
+
+src_prepare() {
+	sed -r \
+		-e 's|^CFLAGS(\s*)=|CFLAGS\1+=|' \
+		-e 's|^LDFLAGS(\s*)=|LDFLAGS\1+=|' \
+		-e "s|^CC=.*|CC=$(tc-getCC) \$(CFLAGS) -Ideps|" \
+		-i Makefile || die "failed to filter Makefile"
```

Comment:
I prefer sed for smaller changes because it’s much simpler to maintain and more tolerant to changes in the source file.